app/javascript/react/screens/App/Plan/helpers.js

```diff
@@ -26,7 +26,8 @@ const processVMTasks = vmTasks => {
       status: task.status,
       options: {},
       cancel_requested: task.options.cancel_requested,
-      source_id: task.source_id
+      source_id: task.source_id,
+      log_available: task.options.virtv2v_wrapper && task.options.virtv2v_wrapper.v2v_log.length > 0
```

Comment:
If `virtv2v_wrapper` is present, will `v2v_log` always be available?

Just checking :)

Reply:
Yes, it should be.
I have tested this on an appliance, and did not have any issues with - `task.options.virtv2v_wrapper && task.options.virtv2v_wrapper.v2v_log`